integration_tests/src/main/python/cache_test.py

```diff
@@ -212,3 +212,26 @@ def test_cache_partial_load(data_gen, enableVectorizedConf):
             .cache()
             .limit(50).select(f.col("b")), enableVectorizedConf
     )
+
+@pytest.mark.parametrize('data_gen', all_gen, ids=idfn)
+@pytest.mark.parametrize('ts_write', ['TIMESTAMP_MICROS', 'TIMESTAMP_MILLIS'])
+@pytest.mark.parametrize('enableVectorized', ['true', 'false'], ids=idfn)
+@allow_non_gpu('CollectLimitExec')
```

Comment:
nit: If we don't have a limit in the code any more a CollectLimitExec is not going to show up.